src/intervals/functions.jl

```diff
@@ -269,3 +269,106 @@ for f in (:log, :log2, :log10, :log1p)
 
         end
 end
+
+```
+This function takes an interval and a conversion specifier as input and gives a string interval containing the interval x.
+    for cs equal to u , it outputs empty, entire and nai in upper case
+    for cs equals to l, it outputs empty, entire and nai in lower case
+    for cs equal to f, it outputs full interval [-Inf,Inf] instead of entire interval.
+    for any other it outputs default string interval.
+```
+function string(x :: Interval{T}, cs :: Char) where T
```

Comment:
I suggest using a symbol instead of / as well as a `Char`,  e.g. `string(x, :u)`.